content/riak/kv/2.0.0/configuring/reference.md

```diff
@@ -196,9 +196,7 @@ executables are stored.</td>
 <tr>
 <td><code>platform_data_dir</code></td>
 <td>The directory in which Riak stores its storage backend data, as well
-as <a href="/riak/kv/2.0.0/learn/concepts/clusters">ring state</a> data,
-<a href="/riak/kv/2.0.0/learn/glossary/#active-anti-entropy-aae">active
-anti-entropy</a> data, and cluster metadata.</td>
+as <a href="../../learn/concepts/clusters">ring state</a> data, <a href="../../learn/glossary/#active-anti-entropy-aae">active anti-entropy</a> data, and cluster metadata.</td>
```

Comment:
The line-width is really inconsistent here... Not sure if we care enough to change it, but I care enough to spend 15 seconds pointing it out, so... (shrug)